### DISCUSSION TITLE
module "github.com/google/docsy/dependencies" not found

### DISCUSSION THREAD

**Discussion by nealkruis** [google/docsy#1504] [Q&A]
I am following the [instructions to add Docsy as a Hugo Module](https://www.docsy.dev/docs/get-started/docsy-as-module/start-from-scratch/), but I'm getting the warning:

`module "github.com/google/docsy/dependencies" not found; either add it as a Hugo Module or store it in "/local/path/to/themes".: module does not exist`

I'm not explicitly getting the dependencies (but they are showing up in `hugo mod graph`). I am listing them under config->module->imports.

Am I doing something wrong?

(I'm currently using v0.4.0 on Mac.)

**Comment by deining** [comment]
> (I'm currently using v0.4.0 on Mac.)

Why are you using outdated v0.4.0 if starting with a site from scratch?

Please show your `go.mod` and the relevant `module` section of your site config file, then its more likely that we can help you out. Also the output of `hugo mod graph`might be helpful.

Or make your whole project publicly available (e.g. on GitHub or so).

**Reply by nealkruis** [reply]
@deining I'm using hugo and docsy as part of a python project that automatically generates sites from scratch. I am reluctant to upgrade the automation as the process is generally working fine with the exception of this warning.

The whole project is available on GitHub: https://github.com/bigladder/lattice

You can see an example of the warning in [Actions](https://github.com/bigladder/lattice/actions/runs/4631087451/jobs/8566201975#step:5:16). The actions generate the entire Hugo project, build, and deploy the website.

Here is the content of `go.mod` generated for my example:

```
module examples/address/docs

go 1.19

require github.com/google/docsy v0.4.0 // indirect
```

Here is the `module` section from the YAML config file:

```
module:
  hugoVersion:
    extended: true
    min: 0.73.0
  imports:
  - path: github.com/google/docsy
    disable: false
  - path: github.com/google/docsy/dependencies
    disable: false
```

And here is the output of `hugo mod graph`:

```
examples/address/docs github.com/google/docsy@v0.4.0
examples/address/docs github.com/google/docsy/dependencies@v0.4.0
```
